docs/websites/ror/host-multiple-rails-apps-with-nginx-puma-postgres-and-rbenv-ubuntu-14-04-trusty-tahr.md

```diff
@@ -15,96 +15,99 @@ contributor:
     link: https://github.com/rickpeyton
 ---
 
-Like any awesome Rails developer you likely have several play apps. But just because these aren't serious production apps doesn't mean that you should have to host them somewhere that forces you to shut them down for 6 hours a day. You can host several Rails apps on a single Linode by utilizing this guide.
+Like any skilled Rails developer, you probably have several play apps. But just because these aren't serious production apps doesn't mean that you should have to host them somewhere that forces you to shut them down for six hours a day. Instead, follow this guide to host several Rails apps on a single Linode and subsequently avert daily shutdowns.
 
 ## Before you begin
 
-This guide provides step by step instructions for hosting multiple Rails apps on Ubuntu 14.04 (Trusty Tahr) with Linode.
+This guide provides step-by-step instructions for hosting multiple Rails apps on Ubuntu 14.04 (Trusty Tahr) with Linode.
 
-Before you begin ensure that you have followed the [Getting Started](https://www.linode.com/docs/getting-started) and [Securing Your Server](https://www.linode.com/docs/security/securing-your-server/) guides.
+Before you begin, ensure that you have followed the [Getting Started](https://www.linode.com/docs/getting-started) and [Securing Your Server](https://www.linode.com/docs/security/securing-your-server/) guides.
 
-This guide assumes that you have created, and are logged in as, a non-root user with sudo privileges. This guide uses the name rick, but update your commands appropriately.
+This guide assumes that you have created and are logged in as a non-root user with sudo privileges. This guide uses the name rick, but you can use whatever name you choose and alter your commands accordingly.
 
 ##Configuring your first Rails app
 
 ### Install Postgres
 
-1.  Update the system and install essentials
+1.  Update the system and install essentials by entering: 
 
         sudo apt-get update
         sudo apt-get upgrade --show-upgraded
         sudo apt-get install -y build-essential libssl-dev libreadline-dev zlib1g-dev git postgresql postgresql-contrib libpq-dev nodejs nginx
 
-2.  Setup a postgres user with the same name as the current user
+2.  Setup a postgres user with the same name as the current user:
 
         sudo -u postgres createuser --superuser $USER
+
+    Log into postgres and set the password for your user, if you wish:
+
         sudo -u postgres psql
-        \password $USER
+        \password username
 
-As this guide assumes you will be using a local Postgres database you may wish to leave the password blank
+    This guide assumes you will be using a local Postgres database; thus, you may wish to leave the password blank.
 
-3.  To exit Postgres
+3.  To exit Postgres, type: 
 
         \q
 
 
 ### Install rbenv
```

Comment:
Put yourself in the shoes of someone looking at multiple ruby environments for the first time ( who would need this guide ). Should they already know what `rbenv` is? If not, explain what it is and why we're installing it.